frontend/database/161_add_missing_public_tags.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO
+    `Tags` (`name`, `public`)
+VALUES
+    ('problemTagAnalysisOfRecurrences', 1),
+    ('problemTagUnformattedInputAndOutput', 1),
+    ('ProblemTagFileSeeking', 1);
```

Comment:
nit:

```suggestion
    ('problemTagFileSeeking', 1);
```

esto evita que hayan errores al momento de validar que este es un tag público. igual en los demás lugares donde se usa esto.

Reply:
Listo